qa/integration-tests/src/test/java/io/camunda/it/client/GetDecisionDefinitionXmlTest.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.it.client;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import io.camunda.qa.util.cluster.TestStandaloneCamunda;
+import io.camunda.zeebe.client.ZeebeClient;
+import io.camunda.zeebe.client.api.command.ProblemException;
+import io.camunda.zeebe.client.api.response.Decision;
+import io.camunda.zeebe.client.api.response.DeploymentEvent;
+import io.camunda.zeebe.qa.util.junit.ZeebeIntegration;
+import io.camunda.zeebe.qa.util.junit.ZeebeIntegration.TestZeebe;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.Random;
+import java.util.concurrent.CompletionException;
+import org.awaitility.Awaitility;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+@ZeebeIntegration
+public class GetDecisionDefinitionXmlTest {
```

Comment:
❓ Just wondering: should we maybe integrate the test cases into the `SearchDecisionDefinitionTest` and rename that one? Could help us save resources used for integration tests. Just a thought, let me know what you think.

Reply:
That makes sense; there’s no need to launch a new test application and Elasticsearch instance just for these couple of tests.

Reply:
I moved the tests to `DecisionQueryTest` I also did the same for `SearchDecisionRequirementsTest` tests which share the same preconditions